CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - Avoid creating new objects on all calls to `WithDeferredSetup` and `SkipContextSetup` in OpenTracing bridge. (#3833)
 - The `New` and `Detect` functions from `go.opentelemetry.io/otel/sdk/resource` return errors that wrap underlying errors instead of just containing the underlying error strings. (#3844)
+- The metric `Export` interface to accepts a `*ResourceMetrics` instead of `ResourceMetrics` (#3853)
```

Comment:
```suggestion
- The `Export` interface in `go.opentelemetry.io/otel/exporters/stdout/stdoutmetric` to accept `*metricdata.ResourceMetrics` instead of `metricdata.ResourceMetrics`. (#3853)
```

I think this makes it easier to understand where the change happened.

Reply:
The interface is in `go.opentelemetry.io/otel/sdk/metric`

Reply:
You're right, thanks!